app/models/group.rb

```diff
@@ -168,6 +168,7 @@ class Group < ActiveRecord::Base # rubocop:disable Metrics/ClassLength
 
   validates_by_schema except: [:logo]
   validates :type, uniqueness: { scope: :parent_id }, if: :static_name
+  validates :parent, presence: true, unless: :root?
```

Comment:
I noticed that this is tautological. `:root?` comes form the awesome_nested_set gem and is defined like this:

https://github.com/collectiveidea/awesome_nested_set/blob/master/lib/awesome_nested_set/model.rb#L129-L131

Reply:
But if there can be only one `root` we can check if `Groop.root` exists.

Reply:
OK I'm not sure anymore if this is the way to go, since now a lot of other specs fail.

Reply:
Changed it again to `unless: layer?`, which seems feasible.